config/200-clusterrole.yaml

```diff
@@ -35,6 +35,9 @@ rules:
   - apiGroups: ["triggers.tekton.dev"]
     resources: ["clustertriggerbindings/status", "eventlisteners/status", "triggerbindings/status", "triggertemplates/status"]
     verbs: ["get", "list", "create", "update", "delete", "patch", "watch"]
+  - apiGroups: ["triggers.tekton.dev"]
```

Comment:
Instead of adding a new rule here, we can just add "triggers" to the rule in [line 33](https://github.com/tektoncd/triggers/blob/master/config/200-clusterrole.yaml#L33) and the status one in the following rule: https://github.com/tektoncd/triggers/blob/master/config/200-clusterrole.yaml#L36